setup-homebrew/main.sh

```diff
@@ -220,8 +227,7 @@ else
         git_retry clone https://github.com/Homebrew/homebrew-cask "${HOMEBREW_CASK_REPOSITORY}"
```

Comment:
You'll need to adjust the condition here to remove the `GITHUB_REPOSITORY` check and add `UPDATE_CASK` if you want to replace `brew tap homebrew/cask` everywhere.